src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -119,13 +119,8 @@ CHIP_ERROR ScenesServer::Init()
         EmberAfStatus status = Attributes::FeatureMap::Get(endpoint, &featureMap);
         if (EMBER_ZCL_STATUS_SUCCESS == status)
         {
-            // Setting NameSupport attribute value to 0x80 if the feature bit is set
-            //  The bit of 7 (0x80) the NameSupport attribute indicates whether or not scene names are supported
-            //
-            //  According to spec, bit 7 (Scene Names) MUST match feature bit 0 (Scene Names)
-            uint8_t nameSupport =
-                (featureMap & to_underlying(Feature::kSceneNames)) ? static_cast<uint8_t>(0x80) : static_cast<uint8_t>(0x00);
-            status = Attributes::NameSupport::Set(endpoint, nameSupport);
+            // According to spec, bit 7 MUST match feature bit 0 (SceneNames)
+            status = Attributes::NameSupport::Set(endpoint, NameSupportBitmap::kSceneNames);
```

Comment:
But ... this is no longer looking at the value of `featureMap`?

Reply:
Right, dropped the check by mistake, it is back now.

Reply:
Why were there no CI failures?  Or were there?

Reply:
The CI didn't run because the restyler had failed, and now it is not running since there are conflicts that I need to address.